cogs/moderation.py

```diff
@@ -293,6 +298,96 @@ async def clear(self, ctx: Context, amount: int) -> None:
         await asyncio.sleep(2.5)
         await message.delete()
 
+    @command()
+    @has_permissions(manage_messages=True)
+    async def shift(self, ctx, count: int, target: TextChannel, copy: bool = False) -> None:
+        """Copy or Move specified messages amount to specified channel"""
+        messages = []
+        async for message in ctx.message.channel.history(limit=count):
+            embed = Embed(description=message.content, color=Color.green())
+            embed.set_author(name=message.author.name, icon_url=message.author.avatar_url)
+            embed.timestamp = message.created_at
+            messages.append(embed)
+
+            if not copy:
+                await message.delete()
+
+        await target.send(f'Message source : {ctx.channel.mention}.')
+
+        for embed in reversed(messages):
+            await target.send(embed=embed)
+
+    @command()
+    @has_permissions(manage_messages=True)
+    async def dm(self, ctx: Context, members: Greedy[Member], *, message: str) -> None:
+        """Dm a List of Specified User from Your Guild."""
+        embed = Embed(
+            title="Notice!",
+            description=message,
+            color=Color.dark_magenta()
+        )
+        embed.set_footer(text=f"From {ctx.guild.name}", icon_url=ctx.guild.icon_url)
+
+        for member in members:
+            await member.send(embed=embed)
```

Comment:
It would be nice if you could also pass in roles and DM everyone in that role like that.
Do note that doing that would mean that the user will need administrator permissions, not just manage messages